packages/plugins/Gfi/src/utils/requestGfiGeoJson.ts

```diff
@@ -0,0 +1,16 @@
+import { Feature as GeoJsonFeature } from 'geojson'
+import { GeoJSON } from 'ol/format'
+
+const writer = new GeoJSON()
+
+/**
+ * Returns features from GeoJSON layer as GeoJSON.
+ */
+export default ({ map, coordinate, layer }): Promise<GeoJsonFeature[]> =>
+  Promise.resolve(
+    map
+      .getFeaturesAtPixel(map.getPixelFromCoordinate(coordinate), {
+        layerFilter: (candidate) => candidate === layer,
```

Comment:
Missing type for `candidate`

Reply:
https://github.com/Dataport/polar/commit/2f0a6586f951dd11a268fbeec22ba2a2119903b3